redfish-exporter/docs/metrics.md

```diff
@@ -0,0 +1,38 @@
+# Metrics Documentation
+
+The AMD Redfish Exporter exposes the following Prometheus-compatible metrics:
+
+## Event Metrics
+
+### `RedFishEvents_received`
+
+- **Type**: Counter
+- **Description**: Total number of Redfish events received
+- **Labels**:
+  - `SourceIP`: IP address of the Redfish server that sent the event
+  - `Severity`: Severity of the Redfish event
+
+### `RedFishEvents_processing_time`
+
+- **Type**: Gauge
+- **Description**: Time taken to process Redfish events in seconds
+- **Labels**:
+  - `SourceIP`: IP address of the Redfish server that sent the event
+  - `Severity`: Severity of the Redfish event
+
```

Comment:
Do we need add `SlurmAPI_failure` and `SlurmAPI_success` metrics that's defined here https://github.com/nod-ai/ADA/blob/main/redfish-exporter/metrics/metrics.go#L39-L53?

Reply:
Added these to `metrics.md`